.env

```diff
@@ -1,2 +1,8 @@
-AUTHENTICATION_TOKEN: 
-JSONWT_SECRET='secret12345678'
\ No newline at end of file
+JWTPRIVATEKEY='anything@epicMAIL'
```

Comment:
This file shouldn't be online. They contain sensitive information which should not be open for people to see/have access.

Reply:
Already addressed